fdbserver/Knobs.h

```diff
@@ -171,7 +171,9 @@ class ServerKnobs : public Knobs {
 	int DD_EXCLUDE_MIN_REPLICAS;
 	bool DD_VALIDATE_LOCALITY;
 	int DD_CHECK_INVALID_LOCALITY_DELAY;
-
+	int64_t DD_SS_FAILURE_VERSIONLAG;
```

Comment:
A comment for these two lags will be helpful for the future us who reads this knob.

Reply:
Will do.